### PR TITLE
Feature/greetings

### PR DESCRIPTION
**Pull request by D4nielJ**
## Hello reviewer :wave: :vulcan_salute: :rocket: 

This PR contains:

- A Ruby on Rails project with react-rails gem
- Adds an endpoint on `v1/greeting` that returns a random string
- Displays react application on root
- A Greeting component that dispatches an action to fetch the random string from the JSON endpoint.

### Thank you for your feedback, is greatly appreciated. :rose:

### CODE REVIEW

File: app/controllers/v1/greetings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class V1::GreetingsController < ApplicationController
+  def index
```
Comment (rloterh) [inline app/controllers/v1/greetings_controller.rb:2]: - [ ] Both are rails and react (V1/greetings) are displaying blank screens. Kindly check and improve upon the same.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Daniel, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/v1/greetings_controller.rb (hunk 1 above)
Comment (D4nielJ) [reply app/controllers/v1/greetings_controller.rb:2]: I'm sorry, I provided the incorrect endpoint, the correct one is `v1/greeting`.

I think that the react components should be rendering correctly aswell, if you encounter an issue with them, try running concurrently `./bin/webpack-dev-server` with `rails server`

**Review by Lameck1 — APPROVED** [review]
Hi, @D4nielJ,

You've done well addressing the concerns in the previous review 👏🏼.
The tasks for this project are now successfully implemented 💯.
All the best with the next project.

You are free to merge this branch :shipit:

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**